src/Fixer/Import/NoUnusedImportsFixer.php

```diff
@@ -171,6 +171,10 @@ private function getNamespaceUseDeclarations(Tokens $tokens, array $useIndexes)
                 continue;
             }
 
+            if (1 === preg_match('/^ (function|const)(.+)$/', $declarationContent, $matches)) {
```

Comment:
I think there must be one or more whitespace after `function|const` to prevent matching `use functionABC;`
Not sure about the `$` here, for example this valid:

```php
<?php

use function
    is_int;
```

(
```php
<?php

use function
 /*  */   is_int;
```

as well :|
)

The first space to match, are we sure there is always one (and not zero or multiple or combined with linebreaks and such)?